documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generating-necessary-inputs.rst

```diff
@@ -0,0 +1,84 @@
+.. _generating-necessary-inputs:
+
+Generate the necessary input files
+#############################################################
+
+The ``sct_compute_compression`` command uses the shape metric of the spinal cord at the level of compression and computes a ratio with non-compressed levels above and below the compression site.
+
+
+To use ``sct_compute_compression``, we need the 3 following input files:
+
+
+1. Spinal cord segmentation
+2. Vertebral labels
+3. Compression labels (``t2_compressed_label-compression.nii.gz``)
+
+You can also get these files by downloading :sct_tutorial_data:`data_normalizing-morphometrics-compression.zip`.
+
+1. Spinal cord segmentation
+----------------------------
+.. code:: sh
+
+   sct_deepseg_sc -i t2_compressed.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+
+:Output files/folders:
+   - ``t2_compressed_seg.nii.gz`` : Spinal cord segmentation.
+
+
+2. Label the spinal cord vertebral levels
+------------------------------------------
+.. code:: sh
+
+   sct_label_vertebrae -i t2_compressed.nii.gz -s t2_compressed_seg.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : The input image file.
+   - ``-s`` : Spinal cord segmentation.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+:Output files/folders:
+   - ``t2_compressed_seg_labeled.nii.gz`` : Spinal cord vertebral labels.
+
+3. Generate spinal cord compression labels
+-------------------------------------------
+
+1. Open the image in FSLeyes. If the image does not have sufficient axial and sagittal resolution, open two individual images (axial image and sagittal image) in FSLeyes.
+
+.. code:: sh
+
+   fsleyes t2_compressed.nii.gz &
+
+2. Look at the clinical data provided with compression information to know at which level the compression(s) are located.
+3. Locate the compression in the sagittal view.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/localizing_compression_sag.png
+   :align: center
```

Comment:
Which direction should I be locating in the sagittal view? Just like, making sure to click on the correct IS slice?

Reply:
Sorry, I do not understand. Can you please clarify what is not clear here?

Reply:
Apologies for the unclear question.  What I mean is:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/c315be97-68b7-44e8-8943-b9e7932cb0d9)

In the sagittal view, should the user be clicking on the center of the spinal cord where there is compression? Or, the anterior or posterior part of the cord? Or, does it not really matter, since the AP and LR coordinates will be fine-tuned when selecting a point in the axial view?

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/fe20ba1e-e98c-4ad9-bd82-afab79aa5179)

Reply:
Ah, I see now. It doesn't matter too much because in the next step, we toggle the slice to determine the one with the greatest compression.

Reply:
Maybe we could update the wording then to say something like:

```suggestion
3. Locate the compression in the sagittal view. (The location does not need to be precise, as we will fine-tune the selection in the next step.)

.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/localizing_compression_sag.png
   :align: center
```

Reply:
yes exactly! we can maybe add more details to the instructions!